server/src/main/java/org/opentosca/toscana/plugins/cloudformation/mapper/JavaRuntimeMapper.java

```diff
@@ -0,0 +1,27 @@
+package org.opentosca.toscana.plugins.cloudformation.mapper;
+
+import org.opentosca.toscana.model.node.custom.JavaRuntime;
+
+import org.slf4j.Logger;
+
+public class JavaRuntimeMapper {
+    private final Logger logger;
+    private static final String DEFAULT_STACK_CONFIG = "64bit Amazon Linux 2017.09 v2.6.6 running Java ";
+    private static final String JAVA_8 = "8";
+
+    public JavaRuntimeMapper(Logger logger) {
+        this.logger = logger;
+    }
+
+    public String mapRuntimeToStackConfig(JavaRuntime javaRuntime) {
+        logger.debug("Map JavaRuntime {}", javaRuntime.getEntityName());
+        String javaVersion;
+        if (javaRuntime.getComponentVersion().isPresent()) {
+            javaVersion = javaRuntime.getComponentVersion().get().replaceAll("1.", "");
```

Comment:
Be warned: `replaceAll` takes a regex. I bet youmeant to write: `replaceAll("1\\.", "");`

But still, reminds me of the Y2K bug (java's latest version is already 1.9) . Consider:
```java
...replaceAll("^.*\\.","")
```

Reply:
Totally missed that